OptimizelySDK/Utils/ConditionEvaluator.cs

```diff
@@ -71,7 +71,7 @@ public bool Evaluate(JToken conditions, UserAttributes userAttributes)
 
             string conditionName = conditions["name"].ToString();
             return userAttributes != null && userAttributes.ContainsKey(conditionName) 
-                && userAttributes[conditionName] == conditions["value"].ToString();
+                && userAttributes[conditionName].ToString() == conditions["value"].ToString();
```

Comment:
This may cause some problem when any value other than string is converted into string. @oakbani